### PR TITLE
Move lock icon position and add additional tooltips to branch list page

### PR DESCRIPTION
**Pull request by william-allspice**
This Pull Request adds missing tool tips for the protected, copy, and rss icons on the branch list page. It also moved protected icon position after the branch name.

### CODE REVIEW

**Comment by lunny** [comment]
Can you post some screenshots?

File: templates/repo/branch/list.tmpl [hunk 1]
```diff
@@ -39,7 +41,7 @@
 									</button>
 								{{end}}
 								{{if .EnableFeed}}
-									<a role="button" class="btn interact-bg tw-p-2" href="{{$.FeedURL}}/rss/branch/{{PathEscapeSegments .DefaultBranchBranch.DBBranch.Name}}">{{svg "octicon-rss"}}</a>
+									<a role="button" class="btn interact-bg tw-p-2" href="{{$.FeedURL}}/rss/branch/{{PathEscapeSegments .DefaultBranchBranch.DBBranch.Name}}" data-tooltip-content="{{ctx.Locale.Tr "rss_feed"}}" data-tooltip-content="{{ctx.Locale.Tr "rss_feed"}}">{{svg "octicon-rss"}}</a>
```
Comment (delvh) [inline templates/repo/branch/list.tmpl]:
```suggestion
									<a role="button" class="btn interact-bg tw-p-2" href="{{$.FeedURL}}/rss/branch/{{PathEscapeSegments .DefaultBranchBranch.DBBranch.Name}}" data-tooltip-content="{{ctx.Locale.Tr "rss_feed"}}">{{svg "octicon-rss"}}</a>
```

**Review by delvh — APPROVED** [review]

**Comment by lunny** [comment]
Non default branch's rss button's tooltips are missed.

**Comment by lunny** [comment]
> Non default branch's rss button's tooltips are missed.

Could you add those class?

**Review by lunny — APPROVED** [review]

**Pull request merged**